README.md

```diff
@@ -1,176 +1,61 @@
+This repository contains code examples that demonstrate how to use the [AWSK SDK's](https://aws.amazon.com/developer/tools/) to interact with [AWS services](https://aws.amazon.com/products).
+
+Many examples are injected into the [AWS Documentation](https://docs.aws.amazon.com).
+
+## How this repository is organized
+Code examples for each language's SDK can be found within the following sub-directories:
+
+| SDK        | folder                                | SDK version | SDK status  |
+|------------|---------------------------------------|-------------|-------------|
+| .NET       | [dotnetv3/](dotnetv3)                 | 3.5+        | ![[]](https://img.shields.io/badge/-GA-blue)           |
+| .NET       | [dotnet/](dotnet)                     | <3.5        | ![[]](https://img.shields.io/badge/-deprecated-red)  |
+| C++        | [cpp/](cpp)                           | 1           | ![[]](https://img.shields.io/badge/-GA-blue)          |
+| Go         | [gov2/](gov2)                         | 2           | ![[]](https://img.shields.io/badge/-GA-blue)          |
+| Go         | [go/](go)                             | 1           | ![[]](https://img.shields.io/badge/-deprecated-red)  |
+| Java       | [javav2/]()                           | 2           | ![[]](https://img.shields.io/badge/-GA-blue)          |
+| Java       | [java/](java)                         | 1           | ![[]](https://img.shields.io/badge/-deprecated-red)  |
+| JavaScript | [javascriptv3/](javascriptv3)         | 3           | ![[]](https://img.shields.io/badge/-GA-blue)          |
+| JavaScript | [javascript/](javascript)             | 2           | ![[]](https://img.shields.io/badge/-deprecated-red)  |
+| Kotlin     | [kotlin/](kotlin)                     |             | ![[]](https://img.shields.io/badge/-preview-brightgreen)       |
+| PHP        | [php/](php)                           | 3           | ![[]](https://img.shields.io/badge/-GA-blue)          |
+| Python     | [python/](python)                     | 3           | ![[]](https://img.shields.io/badge/-GA-blue)          |
+| Ruby       | [ruby/](ruby)                         | 3           | ![[]](https://img.shields.io/badge/-GA-blue)          |
+| Rust       | [rust_dev_preview/](rust_dev_preview) |             | ![[]](https://img.shields.io/badge/-preview-brightgreen) |
+| Swift      | [swift/](swift)                       |             | ![[]](https://img.shields.io/badge/-preview-brightgreen)     |
+
+Within each directory, you will find SDK-specific instructions for understanding and invoking example code.
+
+## Invoking example code
+To invoke this example code, you must have an AWS account. For more information about creating an account, see [AWS Free Tier](https://aws.amazon.com/free/). 
+
+You must also have AWS credentials configured. For steps on using the CLI to configure credentials, see [CLI Configuration basics](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html)
```

Comment:
what if they mighn't have AWS CLI installed, or not want to use it, so we should consider saying somthing like 'Alternatively consult the SDK guide for their chosen language (see SDKs and Toolkits section on https://docs.aws.amazon.com/)

Reply:
Good idea, except this might send them on a hunt with a guarantee to find nothing, because the AWS SDK's all use the same underlying credential management tool. So we might just want to say add `For steps on using the CLI to configure credentials (recommended), see...`